docs/spark/what-is-apache-spark-dotnet.md

```diff
@@ -3,26 +3,55 @@ title: What is .NET for Apache Spark?
 description: Learn about .NET for Apache Spark, a free, open-source, and cross-platform big data analytics framework that takes Spark anywhere you write .NET code.
 author: mamccrea
 ms.topic: overview
-ms.date: 05/06/2019
+ms.date: 09/27/2019
 ---
 
 # What is .NET for Apache Spark?
 
-[Apache Spark](https://spark.apache.org/) is a general-purpose distributed processing engine for analytics over large data sets-typically terabytes or petabytes of data. .NET for Apache Spark makes Apache Spark accessible for developers, bringing the power of Spark to your applications with support for languages you already know.
+[Apache Spark](what-is-spark.md) is a general-purpose distributed processing engine for analytics over large data sets - typically terabytes or petabytes of data. With .NET for Apache Spark, a free, open-source, and cross-platform big data analytics framework, you can now add the power of Apache Spark to your big data applications using languages you already know.
 
-With C# and F#, you can access:
+## Why choose .NET for Apache Spark?
 
-* Dataframe and SparkSQL for working with structured data.
-* Spark Structured Streaming for working with streaming data.
+.NET for Apache Spark provides high performance APIs for using Spark from C# and F#. With C# and F#, you can access:
+
+* DataFrame and SparkSQL for working with structured data
+* Spark Structured Streaming for working with streaming data
+* Spark SQL
+* MLlib
```

Comment:
We don't currently support MLlib (although it may be on the road map). We might want to rephrase this as just machine learning, and could mention ML.NET integration:

```suggestion
* Machine learning integration for faster training and prediction (i.e. use .NET for Apache Spark alongside [ML.NET](http://dot.net/ml))
```